git/config.go

```diff
@@ -107,6 +110,7 @@ func (s *GitConfigSection) ParseValues(valueslines string) {
 		}
 		varname := strings.TrimSpace(split[0])
 
+                //fmt.Printf("%v\n", varname)
 		s.values[varname] = strings.TrimSpace(strings.Join(split[1:], "="))
```

Comment:
Yes, it looks like I forgot to remove one before committing and didn't notice since I rarely use git config after initially setting up my name/email..